.github/workflows/python-app.yml

```diff
@@ -56,13 +55,12 @@ jobs:
       with:
         python-version: "3.10"
     - name: Cache dependencies
-      uses: actions/cache@v2
+      uses: actions/cache@v3
       with:
         path: |
           ~/.cache/pip
         key: ${{ runner.os }}-pip-${{ hashFiles('requirements.txt') }}
-        restore-keys: |
```

Comment:
Could you clarify why restore-keys are removed now?

Reply:
It is no longer needed in v3 based on the docs PTAL https://github.com/actions/cache

Reply:
I do not see where it says that `restore-keys` is obsolete now.

Reply:
I may have understated wrong but based on the examples the restore-keys wasn't added like before but rather it was added as an if statement so I will implement it like that